Plugins/AWSLambdaPackager/Plugin.swift

```diff
@@ -12,11 +12,18 @@
 //
 //===----------------------------------------------------------------------===//
 
-import Dispatch
-import Foundation
 import PackagePlugin
 import Synchronization
 
+#if canImport(FoundationEssentials)
+import FoundationEssentials
+#else
+import struct Foundation.URL
+import class Foundation.ProcessInfo
+import class Foundation.FileManager
+import struct Foundation.ObjCBool
+#endif
```

Comment:
I'm afraid we need full Foundation here. This is a built time dependency. So therefore we don't really care.
```suggestion
import Foundation
```

Reply:
Correct - just realized that when testing the CI compilation (that failed) - I just pushed a change